nuxt.config.ts

```diff
@@ -8,9 +8,9 @@ export default defineNuxtConfig({
   uniform: {
     projectId: process.env.UNIFORM_PROJECT_ID,
     readOnlyApiKey: process.env.UNIFORM_API_KEY,
-    apiHost: process.env.UNIFORM_CLI_BASE_URL,
-    edgeApiHost: process.env.UNIFORM_CLI_BASE_EDGE_URL,
-    outputType: process.env.OUTPUT_TYPE,
+    apiHost: process.env.UNIFORM_CLI_BASE_URL || "https://uniform.app",
+    edgeApiHost: process.env.UNIFORM_CLI_BASE_EDGE_URL || "https://uniform.global",
+    outputType: process.env.OUTPUT_TYPE || 'standard',
```

Comment:
These are already set as default values on the module-level. Did you face any issues with that?

Reply:
Done!